### PR TITLE
modify log in order to be same below

### PR DESCRIPTION
**Pull request by huangyanhong**
Signed-off-by: 黄艳红00139573 <huang.yanhong@zte.com.cn>

**-modify log**

**- modify log and build sucess**

**- build sucess**

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: volume/store/store.go [hunk 1]
```diff
@@ -146,7 +146,7 @@ func (s *VolumeStore) Purge(name string) {
 	v, exists := s.names[name]
 	if exists {
 		if _, err := volumedrivers.ReleaseDriver(v.DriverName()); err != nil {
-			logrus.Errorf("Error dereferencing volume driver: %v", err)
+			logrus.Errorf("Error removing volume driver: %v", err)
```
Comment (ripcurld0) [inline volume/store/store.go]: how about `Error releasing volume driver`?

**Comment by ripcurld0** [comment]
@thaJeztah PTAL

File: volume/store/store.go (hunk 1 above)
Comment (thaJeztah) [reply volume/store/store.go]: Yes, "releasing" would be better; perhaps use the same as is used in `VolumesStore.create()`; https://github.com/moby/moby/blob/4d8598ad0506b29c12632c1b8ed92eb58fc2f0e2/volume/store/store.go#L412-L414

**Review by thaJeztah — CHANGES_REQUESTED** [review]

File: volume/store/store.go (hunk 1 above)
Comment (huangyanhong) [reply volume/store/store.go]: ok , thanks ,I just have a try  the commit process ,but  everyone is so enthusiastic, I will  work harder

**Comment by ripcurld0** [comment]
@huangyanhong do you mind changing this again to:
```go
logrus.WithError(err).WithField("driver", v.DriverName()).Error("Error releasing reference to volume driver") 
```
from
```go
logrus.Errorf("Error releasing volume driver: %v", err)
```

Thanks a lot! 😀

**Comment by huangyanhong** [comment]
@ripcurld0 driver have been deleted, can‘t use the driver again;

**Comment by ripcurld0** [comment]
How about doing it like this:
```go
 	if exists {
 		name := v.DriverName()
 		if _, err := volumedrivers.ReleaseDriver(name); err != nil {
			logrus.WithError(err).WithField("driver", name).Error("Error releasing reference to volume driver") 
 		}
 	} 
```

@thaJeztah WDYT?

**Comment by thaJeztah** [comment]
Looks like that would work

**Review by thaJeztah — APPROVED** [review]
LGTM, thanks!

**Review by cpuguy83 — APPROVED** [review]
LGTM

**Review by tonistiigi — APPROVED** [review]

**Comment by kolyshkin** [comment]
LGTM

**Pull request merged**